### PR TITLE
Enable HIP-583 features on testnet and mainnet (#4868)

### PR DESCRIPTION
**Pull request by stoqnkpL**
Signed-off-by: Stoyan Panayotov <stoyan.panayotov@limechain.tech>

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/configuration/previewnet/bootstrap.properties [hunk 1]
```diff
@@ -3,7 +3,4 @@ contracts.chainId=297
 bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
-autoCreation.enabled=true
-lazyCreation.enabled=true
-cryptoCreateWithAliasAndEvmAddress.enabled=true
\ No newline at end of file
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```
Comment (david-bakin-sl) [inline hedera-node/configuration/previewnet/bootstrap.properties:6]: are we still interested in turning on all sidecars everywhere? (e.g., as seen in #4708) - in that case this line shouldn't be here, it should be in `hedera-node/hedera-mono-services/src/main/resources/bootstrap.properties` @ line 89

Also totally removed from other per-environment files.

If we still want to do that, please see #4708.

**Review by david-bakin-sl — COMMENTED** [review]
I'm not sure what state we want to have for the action sidecar, and all other sidecars, in this release.  #4708 never got approved for merge.  Marking "request changes" so this discussion happens.  And then after that discussion - I'll switch to approve of course.

**Review by david-bakin-sl — CHANGES_REQUESTED** [review]
(Hit wrong button, can't fix it AFAIK.  Sorry: _now_ marking as "request changes".)

**Review by Nana-EC — APPROVED** [review]

File: hedera-node/configuration/previewnet/bootstrap.properties (hunk 1 above)
Comment (Nana-EC) [reply hedera-node/configuration/previewnet/bootstrap.properties:6]: This is 0.35 so it's fine to be on in previewnet. 
There's no change to side cars so things should be fine.
We'll plan to turn that on in 0.36

**Review by dimitar-dinev — APPROVED** [review]

**Review by david-bakin-sl — APPROVED** [review]

File: hedera-node/configuration/previewnet/bootstrap.properties (hunk 1 above)
Comment (david-bakin-sl) [reply hedera-node/configuration/previewnet/bootstrap.properties:6]: ok

**Pull request merged**